src/transaction/log_page_buffer.c

```diff
@@ -8413,6 +8415,11 @@ logpb_restore (THREAD_ENTRY * thread_p, const char *db_fullname, const char *log
   REL_COMPATIBILITY compat;
   int dummy;
 
+  bool is_prev_volheader_restored = false;
+  // *INDENT-OFF*
+  FILEIO_UNLINKED_VOLINFO unlinked_volinfo;
+  // *INDENT-ON*
```

Comment:
it may not be needed?
```suggestion
  FILEIO_UNLINKED_VOLINFO unlinked_volinfo;
```